Resizetizer.NT/Resizer.cs

```diff
@@ -24,12 +24,9 @@ public string GetFileDestination(DpiPath dpi)
 
 		public static string GetFileDestination(SharedImageInfo info, DpiPath dpi, string intermediateOutputPath)
 		{
-			var name = Path.GetFileNameWithoutExtension(info.Filename);
-			var ext = Path.GetExtension(info.Filename);
-
 			var fullIntermediateOutputPath = new DirectoryInfo(intermediateOutputPath);
 
-			var destination = Path.Combine(fullIntermediateOutputPath.FullName, dpi.Path, name + dpi.FileSuffix + ext);
+			var destination = Path.Combine(fullIntermediateOutputPath.FullName, dpi.Path, info.OutputName + dpi.FileSuffix + info.OutputExtension);
```

Comment:
Instead of using the filename directly, allow this to be overridden using the `Link="xxx"` attribute. The `Output*` properties are basically `Alias ?? Filename` types.